mpi4jax/cython/mpi_xla_bridge.pyx

```diff
@@ -24,6 +26,26 @@ from mpi4py.libmpi cimport (
 MPI_STATUS_IGNORE_ADDR = int(<uint64_t>MPI_STATUS_IGNORE)
 
 
+#
+# Error handling
+#
+
+cdef inline int abort_on_error(int ierr, MPI_Comm comm, unicode mpi_op) nogil:
+    cdef int rank
```

Comment:
Can you just put this cdef below the `    if ierr == MPI_SUCCESS:` line? 
Makes it clearer for me.

Reply:
No, `cdef`s must be declared at the start of the function.